src/components/profile-update/profile-update.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import { Form, Button, Card, CardGroup, Container, Col, Row } from 'react-bootstrap';
+import { BrowserRouter as Router, Link } from 'react-router-dom';
+
+import React, { useState } from 'react';
+import axios from 'axios';
+import PropTypes from 'prop-types';
+
+
+
+export function ProfileUpdate(props) {
```

Comment:
I like the fact that you chose to use a functional component. A class component looks like this
```js
class MainView extends React.Component {

  constructor() {
    super();
    this.state = {
    };

  render() {
    return (<></>)
  }
}
```
A functional component looks like the one you've created or you can call it a functional component
```js
export function ProfileUpdate(props) {
  const { a } = props;
  return (<>{a}</>);
}
```

Reply:
I chose the functional component as I thought it hsould be same as create a user in our registration-view, but with axios.PUT instead of POST. At the end managed to get lost in code anyway :/

Reply:
Was that better choice than class component? Could I use state in class component?

Reply:
Try to default to functional components always. That utilizes the new version of react. Class components are old school and in later versions, we shall get rid of all of them once we start using `redux` to manage state